nixos/tests/ntpd-rs.nix

```diff
@@ -0,0 +1,55 @@
+import ./make-test-python.nix ({ lib, ... }:
+{
+  name = "ntpd-rs";
+
+  meta = {
+    maintainers = with lib.maintainers; [ fpletz ];
+  };
+
+  nodes = {
+    client = {
+      services.ntpd-rs = {
```

Comment:
Just a question, are these two independent and different configurations?

Reply:
Yes, the client will query the time from the server. Unfortunately it can't be fully tested because the stratums match and the server cannot reach upstream NTP servers. So this test only checks if both come up with this configuration.